src/type/p5.Font.js

```diff
@@ -622,12 +627,12 @@ export class Font {
       const vA = newVertices[a];
       const vB = newVertices[b];
       // Skip if vertices are too close (degenerate edge)
-      const dist = Math.sqrt(
-        Math.pow(vB.x - vA.x, 2) +
-        Math.pow(vB.y - vA.y, 2) +
-        Math.pow(vB.z - vA.z, 2)
-      );
-      if (dist < 0.0001) continue;
+      // Check if the cross product of edge vectors has sufficient magnitude
```

Comment:
ohh actually your comment made me realize why your earlier solution worked -- these are just *extruded* faces, not the front faces, so as long as the extrude distance is > 0, we just need to check the edge distance.

Sorry for not fully understanding what you were doing before, it's actually ok to go back to just checking `edgeVector.mag()`, but let's leave a comment explaining why we don't really need to check the other edge (it's just the extrude distance)